internal/raft/message/convert.go

```diff
@@ -0,0 +1,971 @@
+package message
+
+import (
+	"github.com/tomarrell/lbadd/internal/compiler/command"
+)
+
+// ConvertCommandToMessage converts a command.Command to a message.Command
```

Comment:
If it returns a `message.Command`, why does it actually return a `message.Message`?

Reply:
I don't see a change here. GitHub again?